### ISSUE TITLE
AirVPN city does not work

### ISSUE THREAD

**Issue by vassoz** [qdm12/gluetun#1358]
### Is this urgent?

No

### Host OS

Ubuntu server 22.04

### CPU arch

x86_64

### VPN service provider

AirVPN

### What are you using to run the container

docker-compose

### What is the version of Gluetun

latest built on 2022-12-31T17:50:58.654Z (commit ea40b84)

### What's the problem 🤔

With a correct CITY (according to the logs) specified the container does not start.

### Share your logs

```plain text
2023-01-17T18:52:40-08:00 ERROR VPN settings: provider settings: server selection: the city specified is not valid: value is not one of the possible choices: value "phoenix", choices available are Alblasserdam, Atlanta, Georgia, Auckland, Belgrade, Berlin, Bern, Brussels, Bucharest, Chicago, Illinois, Dallas, Texas, Frankfurt, Fremont, California, Jacksonville, Florida, Kiev, London, Los Angeles, Madrid, Manchester, Miami, Montreal, Munich, New York City, Oslo, Pennsylvania, Phoenix, Arizona, Prague, Riga, Sao Paulo, Singapore, Sofia, Stockholm, Tallinn, Tokyo, Toronto, Ontario, Uppsala, Vancouver, Vienna, Zurich
```

### Share your configuration

```yml
...
environment:
      VPN_SERVICE_PROVIDER: airvpn
      VPN_TYPE: wireguard
...
      CITY: Phoenix
...
```

**Comment by vassoz** [comment]
Yeah, I forgot to mention that I tried it as well.
Config:
      SERVER_CITIES: Phoenix
Log:
23-01-18T08:42:07-08:00 WARN Shutdown not completed gracefully: VPN settings: provider settings: server selection: the city specified is not valid: value is not one of the possible choices: value "phoenix", choices available are Alblasserdam, Atlanta, Georgia, Auckland, Belgrade, Berlin, Bern, Brussels, Bucharest, Chicago, Illinois, Dallas, Texas, Frankfurt, Fremont, California, Jacksonville, Florida, Kiev, London, Los Angeles, Madrid, Manchester, Miami, Montreal, Munich, New York City, Oslo, Pennsylvania, Phoenix, Arizona, Prague, Riga, Sao Paulo, Singapore, Sofia, Stockholm, Tallinn, Tokyo, Toronto, Ontario, Uppsala, Vancouver, Vienna, Zurich

**Comment by wutr** [comment]
Do any of the other cities work?

Other than that, your config shows no quotes around the variable value but the error does:
`

> the city specified is not valid: value is not one of the possible choices: value **"**phoenix**"**

`
That may just be down to the way the logging works, but I made the mistake before to put a country in quotes and it resulted in the same error.

**Comment by qdm12** [comment]
That was an odd one, but the city name was `Phoenix, Arizona` with a comma (took me a while too). There was also a faulty code line not removing commas from city names coming from airvpn web API, this got fixed just now in 0df68f76d503cf783f39ff12809d980d81722fd3 so now the city name will be `Phoenix Arizona`.
